technic/machines/register/cables.lua

```diff
@@ -122,7 +125,9 @@ function technic.register_cable_plate(nodename, data)
 				return item_place_override_node(itemstack, placer, pointed_thing, node)
 			end
 		else
-			def.groups.not_in_creative_inventory = 1
+		   def.groups.not_in_creative_inventory = 1
+		   def._mcl_blast_resistance = 1
+		   def._mcl_hardness = 0.8
```

Comment:
Minor indent / style
```suggestion
			def.groups.not_in_creative_inventory = 1
			def._mcl_blast_resistance = 1
			def._mcl_hardness = 0.8
```